.github/workflows/gmake.yml

```diff
@@ -4,7 +4,7 @@ on: [push]
 
 jobs:
     build:
-        runs-on: ubuntu-latest
```

Comment:
"ubuntu-latest" fails to install "libhdf5-dev" because one of the dependent packages is missing from the remote server.